internal/codespaces/api/api_test.go

```diff
@@ -115,6 +115,145 @@ func TestListCodespaces_unlimited(t *testing.T) {
 	}
 }
 
+func createFakeSearchReposServer(t *testing.T, searchText string, responseRepos []*Repository) *httptest.Server {
+	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/search/repositories" {
+			t.Fatal("Incorrect path")
+		}
+
+		if r.URL.Query().Get("q") == "" {
+			t.Fatal("Missing 'q' param")
+		}
+
+		q := r.URL.Query().Get("q")
+		expectedQ := fmt.Sprintf("%s in:name", searchText)
+		if q != expectedQ {
+			t.Fatalf("expected q = '%s', got '%s", expectedQ, q)
```

Comment:
Use %q instead of '%s', throughout. (It properly quotes quotation marks.)

Reply:
Oh cool, didn't know that one - thanks!